src/pages/UserProfile.js

```diff
@@ -0,0 +1,80 @@
+import {useState, useEffect} from 'react';
+import {div, h, hr} from 'react-hyperscript-helpers';
+import {PageHeading} from '../components/PageHeading';
+import {Notification} from '../components/Notification';
+import {FormField, FormFieldTypes, FormValidators} from '../components/forms/forms';
+import {User} from '../libs/ajax';
+import {NotificationService} from '../libs/notificationService';
+import { Notifications} from '../libs/utils';
+
+export default function ResearcherProfile(props) {
```

Comment:
Should be `UserProfile(props)`

On a code organization level, we are definitely going to be adding profile-specific components so let's put this in a `user_profile` directory where all of the relevant components can live. If we decide to re-use any of them, we can move them to the shared component directory.